src/Kunstmaan/NodeBundle/EventListener/NodeTranslationListener.php

```diff
@@ -345,7 +345,7 @@ private function ensureUniqueUrl(
             $oldUrl = $translation->getFullSlug();
             $translation->setSlug(
                 $this->slugifier->slugify(
-                    $this->incrementString($translation->getSlug())
+                    static::incrementString($translation->getSlug())
```

Comment:
This method has no reason to be static in the first place. I would suggest changing the method definition instead of this line. It is private anyway so you always need an instantiated object of this class so the static adjective is useless.